webapp/tsconfig.base.json

```diff
@@ -9,7 +9,7 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "importHelpers": true,
-    "target": "es5",
+    "target": "es2020",
```

Comment:
This gets further transpiled later in our build chain, right? I'm just checking since es2020 won't work with the Chrome/WebView versions some of our programs have deployed.

Reply:
Based on [this conversation](https://github.com/medic/cht-core/pull/7999#discussion_r1068613363) I think we _might_ not need to worry about this any more....

Certainly the linked dependency guidelines specify that we only support `Chrome and Firefox latest` on `4.x`.  I think the idea is that the system webview/browser needs to be kept up to date (even on older devices).

Reply:
If we keep `es5` and build, it throws several errors with typescript v4.8 (min required for Angular 15).